oxo.py

```diff
@@ -1,25 +1,49 @@
+
 class OxoBoard:
     def __init__(self):
         """ The initialiser. Initialise any fields you need here. """
-        raise NotImplementedError("TODO: implement __init__")
+
+        ##This is the code for the array board and its spaces
+        self.theboard = [[0,0,0][0,0,0][0,0,0]]
+        self.col = len(self.board)
+        ##Responsible for the number of rows based on the board
+        self.rownumber = len(self.board[0])
 
     def get_square(self, x, y):
-        """ Return 0, 1 or 2 depending on the contents of the specified square. """
+        """ Return the board's state based on the x and y. """
+        return self.theboard[x][y]
         raise NotImplementedError("TODO: implement get_square")
 
     def set_square(self, x, y, mark):
         """ If the specified square is currently empty (0), fill it with mark and return True.
             If the square is not empty, leave it as-is and return False. """
         raise NotImplementedError("TODO: implement set_square")
 
+        ##If self board values is equal to 0, it means the board has an empty space in which a mark can be maade"
+        if self.theboard[x][y] == 0:
+            self.theboard[x][y] = mark
```

Comment:
So far so good -- just needs to `return True` in this case, `else` it needs to `return False`